src/components/media/MediasSelectModal.jsx

```diff
@@ -291,16 +338,290 @@ const MediasSelectModal = ({
+              <BreadcrumbItem
+                isCurrentPage={
+                  idx === queryParams.mediaDirectoryName.split("%2F").length - 1
+                }
+              >
+                <BreadcrumbLink
+                  onClick={(e) => {
+                    e.preventDefault()
+                    setQueryParams((prevState) => ({
+                      ...prevState,
+                      mediaDirectoryName: getMediaDirectoryName(
+                        queryParams.mediaDirectoryName,
+                        { end: idx + 1 }
+                      ),
+                    }))
+                  }}
+                >
+                  {deslugifyDirectory(dir)}
+                </BreadcrumbLink>
+              </BreadcrumbItem>
+            ))
+          : null}
+      </Breadcrumb>
+    </VStack>
+  )
+
+  return (
+    <Modal
+      isOpen
+      onClose={onClose}
+      size="6xl"
+      scrollBehavior="inside"
+      closeOnOverlayClick={false}
+      isCentered
+      overflowY="hidden"
+    >
+      <ModalOverlay />
+      <ModalContent
+        padding="0.5rem"
+        paddingTop="1rem"
+        maxHeight="90%"
+        overflowY="hidden"
+      >
+        {mediaType === "images" && allowExternal ? (
+          <>
+            <ModalHeader>
+              <VStack alignItems="right" gap="1rem">
+                <HStack justifyContent="space-between">
+                  <Text textStyle="h4">{`Add ${mediaRoom.slice(0, -1)}${
+                    fileName ? ` to ${decodeURIComponent(fileName)}` : ""
+                  }`}</Text>
+                  <ModalCloseButton onClick={onClose} position="static" />
+                </HStack>
+                <Text textStyle="body-1">
+                  You can choose from your {pluralMediaLabel}, upload a new{" "}
+                  {singularMediaLabel}, or add {articleLabel}{" "}
+                  {singularMediaLabel}. You can organise your {pluralMediaLabel}{" "}
+                  in Workspace &gt; {titleCase(pluralMediaLabel)}.
+                </Text>
+                <Tabs width="100%" index={activeTab} onChange={handleTabChange}>
+                  <TabList>
+                    <Tab>Add from {pluralMediaLabel}</Tab>
+                    <Tab>Add by url</Tab>
+                  </TabList>
+                </Tabs>
+                {activeTab === 0 && <FolderBreadcrumb />}
+              </VStack>
+            </ModalHeader>
+            <ModalBody>
+              <Tabs width="100%" index={activeTab} onChange={handleTabChange}>
+                <TabPanels minHeight="16.5rem">
+                  <TabPanel>{internalMediaSelect}</TabPanel>
+                  <TabPanel>
+                    <Grid h="100%" templateColumns="1fr 1fr">
+                      <GridItem
+                        borderRight="1px solid"
+                        borderColor="base.divider.medium"
+                        pr="1.5rem"
+                      >
+                        {register && (
```

Comment:
abit confused as to why this `register &&` is required

Reply:
It isn't! removed in 86e0cfe9